data/mods/ssb/scripts.js

```diff
@@ -187,7 +187,7 @@ let BattleScripts = {
 		if (!item.zMove) return;
 		if (item.zMoveUser && !item.zMoveUser.includes(pokemon.template.species)) return;
 		let atLeastOne = false;
-		/**@type {AnyObject?[]} */
+		/**@type {(RequestSubMoveSlot | null)[]} */
```

Comment:
This can `RequestMoveSlot?[]` because `?` means `| null` in `@type`

Reply:
Fixed